### PR TITLE
ER-01-HTML

### PR DESCRIPTION
**Pull request by ChepeGR**
Replicated [example](https://www.figma.com/file/rDDQzjMzAjQL5NYzCQDiXd/Simple-portfolio-(only-HTML)?type=design&node-id=0-1&mode=design&t=M5vFBqj2PSZd3dSn-0)

### CODE REVIEW

**Review by JulianOviedo — CHANGES_REQUESTED** [review]
Why you used `<footer/>` tag many times ? 

Footer usually its placed at the end of the pages with usefull links like social-media (our example). And also by 99% times the webpage have only one footer tag

Also, there should be just one index.html (main file - home in our case) and the other can be just contact.html. Image you have many files and you have to look for and specific one and you have index 1,2,3,4,5,6,7,8, etc haha

image
doesnt looks like the example i provided, please re-check

**Comment by ChepeGR** [comment]
Thank you for the feedback. I made a new PR based on the comments that you made. Let me know if you have any new comments.

File: homeworks/eduardo-griboldi/contact.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Eduardo Garcia Riboldi</title>
+</head>
+
+<body>
+   
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/home.html"> Home</a>
+        <a href="file:///Users/chepegarciariboldi/Desktop/repoFrontEnd/SOLVD-FE-training/homeworks/eduardo-griboldi/contact.html"> Contact</a>
```
Comment (JulianOviedo) [inline homeworks/eduardo-griboldi/contact.html:12]: could be a `<nav/>`

File: homeworks/eduardo-griboldi/contact.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <br/>
+    
+    <h1>Contact Me</h1>
+    <form>
+        <label>Full name: 
+            <input id="First-Name" type="text"  placeholder="Your name..."/> 
+            <input id="Last-Name" type="text" placeholder="Your last name..."/>
```
Comment (JulianOviedo) [inline homeworks/eduardo-griboldi/contact.html:19]: in general its better to use camelCase naming convention for values in attributtes
https://www.techtarget.com/whatis/definition/CamelCase#:~:text=CamelCase%20is%20a%20way%20to,programming%20and%20computer%20naming%20conventions.

File: homeworks/eduardo-griboldi/contact.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            <input id="Last-Name" type="text" placeholder="Your last name..."/> 
+        </label>
+        <br />
+        <label>Phone number: 
+            <input id="Phone-number" type="text" placeholder="Your phone..." />
+        </label>
+        <br />
+        <label for="Country">Country</label>
+
+            <select name="country" id="country">
```
Comment (JulianOviedo) [inline homeworks/eduardo-griboldi/contact.html:28]: here is one of the reasons why we use a convention naming haha.
`Country` its not the same as `country`

**Review by JulianOviedo — CHANGES_REQUESTED** [review]
image

looks bad, again..

image

Also, when you change smth you should request the re-review

**Review by JulianOviedo — CHANGES_REQUESTED** [review]
image
cant see the profile img.
also, fix navigation between pages.. if i click home or contact in the navbar nothing happens

**Pull request closed without merging**